charts/tfy-logs/values.yaml

```diff
@@ -130,6 +138,31 @@ victoria-logs-single:
 ## @section resourceQuota Add a ResourceQuota to enable priority class in a namespace.
 ##
 
+## @section Windows Vector configurations
+##
+windowsVector:
+  ## @param windowsVector.enabled Enable Windows Vector daemon set for Windows node log collection
+  enabled: false
```

Comment:
If we keep it by default enabled - there is no harm as its going to get deployed only on windows machine